src/MongoDB/ReadConcern.c

```diff
@@ -0,0 +1,198 @@
+/*
+  +---------------------------------------------------------------------------+
+  | PHP Driver for MongoDB                                                    |
+  +---------------------------------------------------------------------------+
+  | Copyright 2013-2015 MongoDB, Inc.                                         |
+  |                                                                           |
+  | Licensed under the Apache License, Version 2.0 (the "License");           |
+  | you may not use this file except in compliance with the License.          |
+  | You may obtain a copy of the License at                                   |
+  |                                                                           |
+  | http://www.apache.org/licenses/LICENSE-2.0                                |
+  |                                                                           |
+  | Unless required by applicable law or agreed to in writing, software       |
+  | distributed under the License is distributed on an "AS IS" BASIS,         |
+  | WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  |
+  | See the License for the specific language governing permissions and       |
+  | limitations under the License.                                            |
+  +---------------------------------------------------------------------------+
+  | Copyright (c) 2014-2015 MongoDB, Inc.                                     |
```

Comment:
Copied from our other headers. I suppose the other question is why do we duplicate the dates in these header blocks.